docs/book/src/quick-start.md

```diff
@@ -12,6 +12,7 @@ This Quick Start guide will cover:
 - [go](https://golang.org/dl/) version v1.15+ (kubebuilder v3.0 < v3.1).
 - [go](https://golang.org/dl/) version v1.16+ (kubebuilder v3.1 < v3.3).
 - [go](https://golang.org/dl/) version v1.17+ (kubebuilder v3.3+).
+- [go](https://golang.org/dl/) version v1.18+ (kubebuilder from master).
```

Comment:
The master doc will become to be prod so we need to keep it == prod
Because this makes no sense we have versions per bin
We do not track these changes. 

IMO: It should be ONLY 
> go](https://golang.org/dl/) version v1.18+

Reply:
So, instead of listing the different versions, just specify 1.18?